stacks/product-aws/main.tf

```diff
@@ -45,6 +45,7 @@ provider "helm" {
 module "product-aws" {
   source                  = "../../modules/lead/product-aws"
   cluster_domain          = var.cluster_domain
+  cluster                 = split(".", var.cluster_domain)[0]
```

Comment:
I'd rather just pass this in as a default product var than compute it like this

Reply:
added it to the list of default product vars